DEVELOPER.md

```diff
@@ -3,40 +3,45 @@
 ## Local Installation
 
 1. Clone this repository to your local machine.
-1. User the `helm` chart in `./deploy/twingate-operator`:
-
+2. Use the `helm` chart in `./deploy/twingate-operator`:
    1. Create a custom `values.yaml`:
 
-   ```bash
-   cp ./deploy/twingate-operator/values.yaml ./deploy/twingate-operator/values.local.yaml
-   ```
+       ```bash
+       cp ./deploy/twingate-operator/values.yaml ./deploy/twingate-operator/values.local.yaml
+       ```
 
-   1. Edit the settings (`twingateOperator` specifically) in
-      `./deploy/twingate-operator/values.local.yaml`
-   1. Deploy:
+   2. Edit the settings (`twingateOperator` specifically) in `./deploy/twingate-operator/values.local.yaml`
+   3. Deploy:
 
-   ```bash
-   helm upgrade twop ./deploy/twingate-operator --install --wait -f ./deploy/twingate-operator/values.local.yaml
-   ```
+      ```bash
+      helm upgrade twop ./deploy/twingate-operator --install --wait -f ./deploy/twingate-operator/values.local.yaml
+      ```
 
 ## Development
 
 1. We use [asdf](https://asdf-vm.com/) for version management. Install it and
    then run `asdf install` in the root of this repo to install the correct
    versions of the tools we use.
-1. We use [direnv](https://direnv.net/) to manage environment variables. Install
+2. Copy the local env file and edit the values to match your environment:
+
+    ```bash
+    cp .envrc.local.example .envrc.local
+    ```
+
+3. We use [direnv](https://direnv.net/) to manage environment variables. Install
    it and then run `direnv allow` in the root of this repo to allow it to manage
    your environment variables.
-1. `cp .envrc.local.example .envrc.local` and edit the values to match your
-   environment.
-1. Install [minikube](https://minikube.sigs.k8s.io/docs/start/)
-   1. `brew install minikube`
```

Comment:
This is a macOS-specific instruction (for Homebrew). If we want to include a command in this step, we should probably include similar commands for steps 1 and 3, too.